scripts/ci/libraries/_initialization.sh

```diff
@@ -153,7 +153,7 @@ function initialization::initialize_base_variables() {
 function initialization::initialize_branch_variables() {
     # Default branch used - this will be different in different branches
     export DEFAULT_BRANCH=${DEFAULT_BRANCH="master"}
-    export DEFAULT_CONSTRAINTS_BRANCH=${DEFAULT_CONSTRAINTS_BRANCH="constraints-master"}
+    export DEFAULT_CONSTRAINTS_BRANCH=${DEFAULT_CONSTRAINTS_BRANCH="constraints-pipcheck"}
```

Comment:
How is `constraints-pipcheck` generated currently?

Reply:
Manually - it's just to test upgrade-first and test-after. Once we make it works and add pip-check it will not be needed.